components/layout/layout_impl.rs

```diff
@@ -237,7 +242,7 @@ impl Layout for LayoutThread {
     )]
     fn query_content_box(&self, node: TrustedNodeAddress) -> Option<UntypedRect<Au>> {
```

Comment:
Could you document this function and `query_content_boxes` below?

In particular: 1. which node's content box? 2. in which node's coordinate space? 3. Is the coordinate space physical or logical?

I'm also wondering if the functions should be renamed, since content box typically indicate that transforms and scroll offsets are not considered. But I'll see after the above documentation is added.

Reply:
1. Node passed as parameter
2. Viewport's coordinate space
3. Physical one

I suppose `query_bounding_box` or maybe `query_bounding_content_box` is the more accurate name.